devices/legrand.js

```diff
@@ -117,23 +117,23 @@ module.exports = [
         onEvent: readInitialBatteryState,
     },
     {
-        zigbeeModel: [' Double gangs remote switch', 'Double gangs remote switch'],
+        zigbeeModel: [' Double gangs remote switch\u0000\u0000\u0000\u0000'],
```

Comment:
Why has `Double gangs remote switch` been removed here?

Reply:
Because external converter doesn't work with it.
I think it's possible to use old variant, but i don't have device with old firmware version to check it.